workers/loc.api/queue/helpers/utils.js

```diff
@@ -29,7 +29,7 @@ const _checkAndCreateDir = async (dirPath) => {
       } catch (errBasePath) {
         if (errBasePath.code === 'EACCES' && isElectronjsEnv) {
           await chmod(basePath, '766')
-        } else throw errBasePath
+        }
```

Comment:
If removing the error, how would be able to log it?

Reply:
I think in this case we should try to create a folder for csv files and then if `mkdir` can't be executed successfully that `mkdir` will throw en error and it would be logged